Update README.md

MohamedOmara-cse opened the pull request.

MohamedOmara-cse committed:
Update README.md

mostafa-hashhash commented:
please keep it in a separate line, just add 2 spaces at the end of the previous line
![image](https://user-images.githubusercontent.com/23399797/106368362-ac59d000-6351-11eb-82e4-090abec9e05b.png)

sorry for the inconvenience and thanks in advance

MohamedOmara-cse committed:
Update README.md

I added one more course (Udacity intro course) to resources in Operating System section
